build: rename (#409)
rune/rune: docs/src/pages/faq.md

In case you want other devs to help you playtest your game, you can [upload your game](/docs/publishing/publishing-your-game) to get a playtest link that you can share with them. This link will work even before the game has been reviewed.

### Can I Upload My Game to Other Platforms Than Dusk? {#can-i-upload-my-game-to-other-platforms-than-dusk}

Definitely. It's your game and you can do whatever you like with it.

### Can I Upload More Than One Game to Dusk? {#can-i-upload-more-than-one-game-to-dusk}

Yes! You can upload as many games as you like to Dusk.

### Can I Remove My Game From Dusk? {#can-i-remove-my-game-from-dusk}

Yes, you're always free to remove your game whenever you'd like. It's your game.

> ### How do I convert my Rune game project to the Dusk SDK? {#how-to-convert-to-dusk}
> 
> Rune was the name previously used for the SDK and service. Your current projects should be updated to use the newer renamed versions of the Games SDK, tools and configuration files.  
> 
> To make the transition as easy as possible, you can follow [this guide](/docs/move-to-dusk) which will help you automatically convert your projects.

Reviewer: ```suggestion
### How do I update my game to use the Dusk SDK instead of the Rune SDK? {#how-to-convert-to-dusk}

You can follow [this guide](/docs/move-to-dusk), which has a tool that automatically converts your game to use the Dusk SDK.
```
